Dockerfile

```diff
@@ -1,31 +1,7 @@
 ARG IMAGE_TAG
 
 # ------ Base -----
-FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04 as base_baal
-
-RUN mkdir -p /app/baal
-RUN apt-get update && apt-get install -y --no-install-recommends \
-  bzip2 \
-  g++ \
-  git \
-  graphviz \
-  libgl1-mesa-glx \
-  libhdf5-dev \
-  openmpi-bin \
-  wget && \
-  rm -rf /var/lib/apt/lists/*
-
-# Install Conda
-ENV CONDA_DIR /opt/conda
-ENV PATH $CONDA_DIR/bin:$PATH
-
-RUN wget --quiet --no-check-certificate https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-latest-Linux-x86_64.sh -f -b -p $CONDA_DIR && \
-    rm Miniconda3-latest-Linux-x86_64.sh && \
-    echo export PATH=$CONDA_DIR/bin:"$PATH" > /etc/profile.d/conda.sh
-RUN conda config --append channels conda-forge
-RUN conda install -y h5py
```

Comment:
Is h5py already included in pytorch image?

Reply:
so changing pytorch image upgrade already containes PL?

Reply:
@parmidaatg PL is installed via PIP from `requirements.txt`

Reply:
LOL sorry abt that =)) +1